src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -422,7 +422,13 @@ Whether Harvesting (OAI) service is enabled
         /**
          * Shibboleth affiliation attribute which holds information about the affiliation of the user (e.g. ou)
          */
-        ShibAffiliationAttribute
+        ShibAffiliationAttribute,
+        /*Convert shibboleth AJP attributes from ISO-8859-1 to UTF-8 */
+        ShibAttributeCharacterSetConversionEnabled,
```

Comment:
ShibAttributeCharacterSetConversionEnabled should be documented in https://github.com/IQSS/dataverse/blob/develop/doc/sphinx-guides/source/installation/config.rst

Reply:
I finished the documentation and cleaned up code formatting. I hope this will be OK.